src/test/java/org/broadinstitute/consent/http/db/DarCollectionDAOTest.java

```diff
@@ -48,24 +46,26 @@ class DarCollectionDAOTest extends DAOTestHelper {
   private void generateDatasetElectionForCollection(DarCollection collection) {
     DataAccessRequest dar = collection.getDars().values().stream()
         .filter(d -> !d.getElections().isEmpty()).findFirst().orElse(null);
+    assertNotNull(dar);
```

Comment:
If you use `orElseThrow()` you don't need `assertNotNull()`
```suggestion
        .filter(d -> !d.getElections().isEmpty()).findFirst().orElseThrow();
```